### PR TITLE
resources/*.html: change to 2-space indentation #7352

### PR DESCRIPTION
**Pull request by sujeet14108**
Fixes partially #7352
All files in `main/resources/*.html`

### CODE REVIEW

File: src/main/resources/feedbackQuestionConstSumEditFormTemplate.html [hunk 1]
```diff
@@ -1,109 +1,109 @@
...
+          <div data-toggle="tooltip" data-placement="top"
+              title="${Const.Tooltips.FEEDBACK_QUESTION_CONSTSUMPOINTS}">
+            <div class="col-xs-4 padding-0 col-sm-4">
+              <input type="number" disabled class="form-control width-100-pc pointsBox"
+                  name="${Const.ParamsNames.FEEDBACK_QUESTION_CONSTSUMPOINTS}"
+                  id="constSumPoints-${questionNumber}" value="${constSumPoints}" min="1"
+                  step="1" onChange="updateConstSumPointsValue(${questionNumber})">
+            </div>
+            <div class="col-xs-6 padding-0">
+              <label class="margin-top-7px padding-left-7px">in
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionConstSumEditFormTemplate.html]: It looks weird for the `in` to be here. Move it to the same line as `total`.

File: src/main/resources/feedbackQuestionConstSumEditFormTemplate.html [hunk 2]
```diff
@@ -1,109 +1,109 @@
...
-                </div>
+        <div class="col-xs-12 margin-bottom-15px padding-left-35px"
+            id="constSumOption_Option-${questionNumber}"${optionDisplay}>
+          <div class="col-xs-1">
+            <input type="radio" id="constSumPointsPerOption-${questionNumber}"
+                name="${Const.ParamsNames.FEEDBACK_QUESTION_CONSTSUMPOINTSPEROPTION}"
+                value="true"${perOptionChecked}>
+          </div>
+          <div data-toggle="tooltip" data-placement="top"
+            title="${Const.Tooltips.FEEDBACK_QUESTION_CONSTSUMPOINTSFOREACHOPTION}">
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionConstSumEditFormTemplate.html]: Add two spaces here.

File: src/main/resources/feedbackQuestionConstSumResultStatsRecipientTemplate.html [hunk 3]
```diff
@@ -1,31 +1,31 @@
...
-                    ${fragments}
-                </tbody>
-            </table>
-        </div>
+  </div>
+  <div class="row">
+    <div class="col-sm-12">
+      <table class="table table-bordered table-responsive margin-0">
+        <thead>
+        <tr>
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionConstSumResultStatsRecipientTemplate.html]: Need to indent more here.

File: src/main/resources/feedbackQuestionConstSumResultStatsRecipientTemplate.html [hunk 4]
```diff
@@ -1,31 +1,31 @@
...
+          <td class="button-sort-none toggle-sort">Team
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionConstSumResultStatsRecipientTemplate.html]: Almost the whole `td` entries look weird. Can tidy up?
```html
<td>
  Team
  <span></span>
</td>
```

File: src/main/resources/feedbackQuestionContribResultStatsStudentViewTemplate.html [hunk 5]
```diff
@@ -1,34 +1,34 @@
...
+  <div class="panel-heading">
+    Comparison of work distribution ${contribAdditionalInfo}
+  </div>
+  <table class="table table-striped">
+    <tbody>
+      <tr>
+        <td>
+          <strong>My view:</strong>
+        </td>
+        <td><span class="text-muted"><strong>of me:</strong></span>
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionContribResultStatsStudentViewTemplate.html]: The `span` tag should be on the next line.

File: src/main/resources/feedbackQuestionContribResultStatsTemplate.html [hunk 6]
```diff
@@ -1,38 +1,38 @@
...
-        </div>
+    <div class="col-sm-3 pull-right">
+      [<a href="/instructorHelp.jsp#faq7a" target="_blank" rel="noopener noreferrer" id="interpret_help_link">How do I interpret/use these values?</a>]
     </div>
+  </div>
+  <div class="row">
+    <div class="col-sm-12">
+      <table class="table table-bordered table-responsive margin-0">
+        <thead>
+        <tr>
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionContribResultStatsTemplate.html]: Needs more indentation here.

File: src/main/resources/feedbackQuestionContribResultStatsTemplate.html [hunk 7]
```diff
@@ -1,38 +1,38 @@
...
+          <td class="button-sort-ascending toggle-sort" id="button_sortteamname">Team
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionContribResultStatsTemplate.html]: Similarly, tidy up the `td` entries here.

File: src/main/resources/feedbackQuestionMcqEditFormOptionFragment.html [hunk 8]
```diff
@@ -1,19 +1,19 @@
 <div class="margin-bottom-7px" id="mcqOptionRow-${i}-${questionNumber}">
-    <div class="input-group width-100-pc">
-        <span class="input-group-addon">
-            <input type="radio" class="disabled_radio" disabled>
+  <div class="input-group width-100-pc">
+    <span class="input-group-addon">
+      <input type="radio" class="disabled_radio" disabled>
+    </span>
+    <input  class="form-control" type="text" disabled
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionMcqEditFormOptionFragment.html]: Remove the extraneous whitespace between `input` and `class`.

File: src/main/resources/feedbackQuestionMcqEditFormTemplate.html [hunk 9]
```diff
@@ -1,57 +1,57 @@
...
-                        disabled
-                        id="generateMcqOptionsCheckbox-${questionNumber}"
-                        ${checkedGeneratedOptions}
-                        onchange="toggleMcqGeneratedOptions(this,${questionNumber})">
-                    Or, generate options from the list of all
-                </span>
+              <input type="checkbox" name="${Const.ParamsNames.FEEDBACK_QUESTION_MCQOTHEROPTIONFLAG}"
+                  id="${Const.ParamsNames.FEEDBACK_QUESTION_MCQOTHEROPTIONFLAG}-${questionNumber}"
+                  ${checkedOtherOptionEnabled} onchange="toggleMcqOtherOptionEnabled(this, ${questionNumber})">
+                Add 'Other' option (Allows respondents to type in their own answer)
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionMcqEditFormTemplate.html]: Remove two spaces from here.

File: src/main/resources/feedbackQuestionMcqEditFormTemplate.html [hunk 10]
```diff
@@ -1,57 +1,57 @@
...
-        <input type="hidden"
-               id="${Const.ParamsNames.FEEDBACK_QUESTION_GENERATEDOPTIONS}-${questionNumber}"
-               name="${Const.ParamsNames.FEEDBACK_QUESTION_GENERATEDOPTIONS}"
-               value="${generateOptionsForValue}">
+      </div>
+    </div>
+
+    <input type="hidden" name="${Const.ParamsNames.FEEDBACK_QUESTION_NUMBEROFCHOICECREATED}"
+      id="${Const.ParamsNames.FEEDBACK_QUESTION_NUMBEROFCHOICECREATED}-${questionNumber}"
+      value="${numOfMcqChoices}">
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionMcqEditFormTemplate.html]: Add two spaces to the above two lines.

File: src/main/resources/feedbackQuestionMcqSubmissionFormOptionFragment.html [hunk 11]
```diff
@@ -1,11 +1,11 @@
...
-            <label class="bold-label"><input type="radio"
-                name="${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}-${questionIndex}-${responseIndex}"
-                ${disabled}
-                ${checked}
-                value="${mcqChoiceValue}"> ${mcqChoiceValue}</label>
-        </div>
-    </td>
+  <td>
+    <div class="radio">
+      <label class="bold-label"><input type="radio"
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionMcqSubmissionFormOptionFragment.html]: This block looks weird. Can tidy up?
```html
<label>
  <input>
  ${mcqChoiceValue}
</label>

File: src/main/resources/feedbackQuestionMcqSubmissionFormOtherOptionFragment.html [hunk 12]
```diff
@@ -1,16 +1,16 @@
...
-                   value="${mcqOtherOptionAnswer}">
-        </div>
-     </td>
- </tr>
+  <td>
+    <div class="radio">
+      <label class="bold-label">
+        <input type="radio"
+            name="${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}-${questionIndex}-${responseIndex}"
+            ${disabled} ${checked} value="${mcqChoiceValue}" data-text="otherOptionText"> Other
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionMcqSubmissionFormOtherOptionFragment.html]: Not your fault, but let's move `Other` to the next line.

File: src/main/resources/feedbackQuestionMsqEditFormOptionFragment.html [hunk 13]
```diff
@@ -1,19 +1,19 @@
 <div class="margin-bottom-7px" id="msqOptionRow-${i}-${questionNumber}">
-    <div class="input-group width-100-pc">
-        <span class="input-group-addon">
-            <input type="checkbox" class="disabled_radio" disabled>
+  <div class="input-group width-100-pc">
+    <span class="input-group-addon">
+      <input type="checkbox" class="disabled_radio" disabled>
+    </span>
+    <input  class="form-control" type="text" disabled
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionMsqEditFormOptionFragment.html]: Remove the extraneous whitespace between `input` and `class`.

File: src/main/resources/feedbackQuestionMsqEditFormTemplate.html [hunk 14]
```diff
@@ -1,58 +1,58 @@
...
-                        </label>
-                    </div>
-                </div>
-            </div>
+          <div class="checkbox">
+            <label class="bold-label">
+              <input type="checkbox" name="${Const.ParamsNames.FEEDBACK_QUESTION_MSQOTHEROPTIONFLAG}"
+                  id="${Const.ParamsNames.FEEDBACK_QUESTION_MSQOTHEROPTIONFLAG}-${questionNumber}"
+                  ${checkedOtherOptionEnabled} onchange="toggleMsqOtherOptionEnabled(this, ${questionNumber})">
+                Add 'Other' option (Allows respondents to type in their own answer)
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionMsqEditFormTemplate.html]: Remove two spaces from here.

File: src/main/resources/feedbackQuestionMsqEditFormTemplate.html [hunk 15]
```diff
@@ -1,58 +1,58 @@
...
         </div>
+      </div>
+    </div>
 
-        <input type="hidden" name="${Const.ParamsNames.FEEDBACK_QUESTION_NUMBEROFCHOICECREATED}"
-            id="${Const.ParamsNames.FEEDBACK_QUESTION_NUMBEROFCHOICECREATED}-${questionNumber}"
-            value="${numOfMsqChoices}">
+    <input type="hidden" name="${Const.ParamsNames.FEEDBACK_QUESTION_NUMBEROFCHOICECREATED}"
+      id="${Const.ParamsNames.FEEDBACK_QUESTION_NUMBEROFCHOICECREATED}-${questionNumber}"
+      value="${numOfMsqChoices}">
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionMsqEditFormTemplate.html]: Add two spaces to the above two lines.

File: src/main/resources/feedbackQuestionMsqSubmissionFormOptionFragment.html [hunk 16]
```diff
@@ -1,11 +1,11 @@
...
-            <label class="bold-label"><input type="checkbox"
-                    name="${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}-${questionIndex}-${responseIndex}"
-                    ${disabled}
-                    ${checked}
-                    value="${msqChoiceValue}"> ${msqChoiceText}</label>
-        </div>
-    </td>
+  <td>
+    <div class="checkbox">
+      <label class="bold-label"><input type="checkbox"
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionMsqSubmissionFormOptionFragment.html]: Similarly, tidy up this block.
```html
<label>
  <input>
</label>
```

File: src/main/resources/feedbackQuestionMsqSubmissionFormOtherOptionFragment.html [hunk 17]
```diff
@@ -1,16 +1,16 @@
...
-                   id="${Const.ParamsNames.FEEDBACK_QUESTION_MSQ_ISOTHEROPTIONANSWER}-${questionIndex}-${responseIndex}"
-                   value="${msqOtherOptionAnswer}">
-        </div>
-     </td>
+  <td>
+    <div class="checkbox">
+      <label class="bold-label">
+        <input type="checkbox"
+            name="${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}-${questionIndex}-${responseIndex}"
+            ${disabled} ${checked} value="${msqChoiceValue}" data-text="msqOtherOptionText"> Other
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionMsqSubmissionFormOtherOptionFragment.html]: Again, `Other` should be on the next line.

File: src/main/resources/feedbackQuestionMsqSubmissionFormOtherOptionFragment.html [hunk 18]
```diff
@@ -1,16 +1,16 @@
...
+      </label>
+      <input class="form-control margin-top-2px" type="text" name="msqOtherOptionText-${questionIndex}-${responseIndex}" id="msqOtherOptionText-${questionIndex}-${responseIndex}"
+          ${text-disabled} value="${msqChoiceValue}">
+      <input type="hidden" name="${Const.ParamsNames.FEEDBACK_QUESTION_MSQ_ISOTHEROPTIONANSWER}-${questionIndex}-${responseIndex}"
+          id="${Const.ParamsNames.FEEDBACK_QUESTION_MSQ_ISOTHEROPTIONANSWER}-${questionIndex}-${responseIndex}"
+          value="${msqOtherOptionAnswer}">
+    </div>
+   </td>
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionMsqSubmissionFormOtherOptionFragment.html]: Remove one space from here. Also one space from the `</tr>` tag following this.

File: src/main/resources/feedbackQuestionNumScaleEditFormTemplate.html [hunk 19]
```diff
@@ -1,31 +1,31 @@
...
-                </div>
-            </div>
-            <br>
-            <div class="row">
-                <div class="col-sm-12">
-                    <span id="numScalePossibleValues-${questionNumber}">${possibleValues}</span>
-                </div>
-            </div>
+      <div class="row">
+        <div class="col-sm-4" data-toggle="tooltip" data-placement="top" title="${Const.ToolTips.FEEDBACK_QUESTION_NUMSCALE_MIN}">Minimum value:
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionNumScaleEditFormTemplate.html]: Move `Minimum value:` to the next line.

File: src/main/resources/feedbackQuestionNumScaleEditFormTemplate.html [hunk 20]
```diff
@@ -1,31 +1,31 @@
...
+          <input disabled type="number" class="form-control minScaleBox" id="minScaleBox-${questionNumber}"
+              name="${Const.ParamsNames.FEEDBACK_QUESTION_NUMSCALE_MIN}"
+              value="${minScale}" onChange="updateNumScalePossibleValues(${questionNumber})">
         </div>
+        <div class="col-sm-4" data-toggle="tooltip" data-placement="top" title="${Const.ToolTips.FEEDBACK_QUESTION_NUMSCALE_STEP}">Increment:
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionNumScaleEditFormTemplate.html]: Move `Increment:` to the next line.

File: src/main/resources/feedbackQuestionNumScaleEditFormTemplate.html [hunk 21]
```diff
@@ -1,31 +1,31 @@
...
+          <input disabled type="number" class="form-control stepBox" id="stepBox-${questionNumber}"
+              name="${Const.ParamsNames.FEEDBACK_QUESTION_NUMSCALE_STEP}"
+              value="${step}" min="0.001" step="0.001"
+              onChange="updateNumScalePossibleValues(${questionNumber})">
+        </div>
+        <div class="col-sm-4" data-toggle="tooltip" data-placement="top" title="${Const.ToolTips.FEEDBACK_QUESTION_NUMSCALE_MAX}">Maximum value:
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionNumScaleEditFormTemplate.html]: Move `Maximum value:` to the next line.

File: src/main/resources/feedbackQuestionNumScaleResultStatsTemplate.html [hunk 22]
```diff
@@ -1,33 +1,33 @@
...
-                ${statsFragments}
-            </table>
-        </div>
+  </div>
+  <div class="row">
+    <div class="col-sm-12">
+      <table class="table table-bordered table-responsive margin-0">
+        <thead>
+          <tr>
+            <td class="button-sort-ascending toggle-sort" id="button_sortreceiverteam">Team
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionNumScaleResultStatsTemplate.html]: Similarly, tidy up all the `td` entries.

File: src/main/resources/feedbackQuestionNumScaleResultStatsTemplateWithSelfResponse.html [hunk 23]
```diff
@@ -1,36 +1,36 @@
...
-    <div class="row">
-        <div class="col-sm-4 text-color-gray">
-            <strong>
-                 ${summaryTitle}
-            </strong>
-        </div>
+  <div class="row">
+    <div class="col-sm-4 text-color-gray">
+      <strong>
+         ${summaryTitle}
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionNumScaleResultStatsTemplateWithSelfResponse.html]: Remove one space from here.

File: src/main/resources/feedbackQuestionNumScaleResultStatsTemplateWithSelfResponse.html [hunk 24]
```diff
@@ -1,36 +1,36 @@
...
-                </thead>
-                ${statsFragments}
-            </table>
-        </div>
+  </div>
+  <div class="row">
+    <div class="col-sm-12">
+      <table class="table table-bordered table-responsive margin-0">
+        <thead>
+        <tr>
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionNumScaleResultStatsTemplateWithSelfResponse.html]: Need more indentation here.

File: src/main/resources/feedbackQuestionNumScaleResultStatsTemplateWithSelfResponse.html [hunk 25]
```diff
@@ -1,36 +1,36 @@
...
+          <td class="button-sort-ascending toggle-sort" id="button_sortreceiverteam">Team
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionNumScaleResultStatsTemplateWithSelfResponse.html]: Similarly, tidy up all the `td` entries here.

File: src/main/resources/feedbackQuestionNumScaleSubmissionFormTemplate.html [hunk 26]
```diff
@@ -1,10 +1,10 @@
 <div class="col-sm-3 mobile-no-padding"><input type="number" class="numScaleAnswerBox form-control col-sm-2" ${disabled} onChange="validateNumScaleAnswer(${questionIndex}, ${responseIndex})"
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionNumScaleSubmissionFormTemplate.html]: The `input` tag should be on the next line.

File: src/main/resources/feedbackQuestionRankEditFormOptionFragment.html [hunk 27]
```diff
@@ -1,16 +1,16 @@
...
-                class="constSumOptionTextBox" value="${rankOptionValue}">
-        <span class="input-group-btn">
-            <button class="btn btn-default removeOptionLink" type="button" id="rankRemoveOptionLink"
-                    onclick="removeRankOption(${i},${questionNumber})"
-                    style="display:none" tabindex="-1">
-                <span class="glyphicon glyphicon-remove">
-                </span>
-            </button>
+  <div class="input-group width-100-pc">
+    <input  class="form-control" type="text" disabled
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionRankEditFormOptionFragment.html]: Remove the extra whitespace between `input` and `class`.

File: src/main/resources/feedbackQuestionRankRecipientsEditFormTemplate.html [hunk 28]
```diff
@@ -1,13 +1,13 @@
...
-        <div class="checkbox"  data-toggle="tooltip" data-placement="top" data-container="body" title="Ticking this will allow response givers to give the same rank to multiple recipients">
-            <label class="bold-label">
-                <input type="checkbox" name="${Const.ParamsNames.FEEDBACK_QUESTION_RANKISDUPLICATESALLOWED}"
-                    id="${Const.ParamsNames.FEEDBACK_QUESTION_RANKISDUPLICATESALLOWED}-${questionNumber}"
-                    ${areDuplicatesAllowedChecked}>
-                Allow response giver to give the same rank to multiple options
-            </label>
-        </div>
+  <div class="col-sm-12">
+    <div class="checkbox"  data-toggle="tooltip" data-placement="top" data-container="body" title="Ticking this will allow response givers to give the same rank to multiple recipients">
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionRankRecipientsEditFormTemplate.html]: Remove the extra whitespace between `"checkbox"` and `data-toggle`.

File: src/main/resources/feedbackQuestionRankResultStatsTemplate.html [hunk 29]
```diff
@@ -1,28 +1,28 @@
...
-                    ${fragments}
-                </tbody>
-            </table>
-        </div>
+  </div>
+  <div class="row">
+    <div class="col-sm-12">
+      <table class="table table-bordered table-responsive margin-0">
+        <thead>
+        <tr>
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionRankResultStatsTemplate.html]: Need more indentation here.

File: src/main/resources/feedbackQuestionRankResultStatsTemplate.html [hunk 30]
```diff
@@ -1,28 +1,28 @@
...
+          <td class="button-sort-ascending toggle-sort" id="button_sortteamname" style="width: 35%;">${optionRecipientDisplayName}
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionRankResultStatsTemplate.html]: Similarly, tidy up all the `td` entries here.

File: src/main/resources/feedbackQuestionRankSubmissionFormOptionFragment.html [hunk 31]
```diff
@@ -1,11 +1,11 @@
...
-        <select class="form-control" ${disabled}
-                id="${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}-${questionIndex}-${responseIndex}-${optionIdx}"
-                name="${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}-${questionIndex}-${responseIndex}"
-                onChange="updateRankMessageQn('${questionIndex}')">
-            ${options}
-        </select>
-    </div>
+  <label class="col-sm-2 control-label" ${rankOptionVisibility}>${rankOptionValue}</label>
+  <div class="col-sm-3">
+  <select class="form-control" ${disabled}
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionRankSubmissionFormOptionFragment.html]: Needs more indentation here.

File: src/main/resources/feedbackQuestionRubricEditFormTemplate.html [hunk 32]
```diff
@@ -1,38 +1,38 @@
...
 <div class="row">
-    <div class="col-sm-6 align-left">
-        <a class="btn btn-xs btn-primary" id="rubricAddSubQuestionLink-${questionIndex}" onclick="addRubricRow(${questionIndex})"><span class="glyphicon glyphicon-arrow-down"> </span> add row</a>
-    </div>
-    <div class="col-sm-6 align-right">
-        <a class="btn btn-xs btn-primary" id="rubricAddChoiceLink-${questionIndex}" onclick="addRubricCol(${questionIndex})">add column <span class="glyphicon glyphicon-arrow-right"></span></a>
-    </div>
-    <br>
+  <div class="col-sm-6 align-left">
+    <a class="btn btn-xs btn-primary" id="rubricAddSubQuestionLink-${questionIndex}" onclick="addRubricRow(${questionIndex})"><span class="glyphicon glyphicon-arrow-down"> </span> add row</a>
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionRubricEditFormTemplate.html]: The line's too long and has too many different HTML tags. Tidy up as such:
```html
<a>
  <span></span> add row
</a>
```

File: src/main/resources/feedbackQuestionRubricEditFormTemplate.html [hunk 33]
```diff
@@ -1,38 +1,38 @@
...
+  </div>
+  <div class="col-sm-6 align-right">
+    <a class="btn btn-xs btn-primary" id="rubricAddChoiceLink-${questionIndex}" onclick="addRubricCol(${questionIndex})">add column <span class="glyphicon glyphicon-arrow-right"></span></a>
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionRubricEditFormTemplate.html]: Similarly here:
```html
<a>
  add column <span></span>
</a>
```

File: src/main/resources/feedbackQuestionRubricSubmissionFormBodyFragment.html [hunk 34]
```diff
@@ -1,3 +1,3 @@
 <td class="col-md-1">
-    <input class="overlay" type="radio" ${disabled} id="${Const.ParamsNames.FEEDBACK_QUESTION_RUBRICCHOICE}-${questionIndex}-${responseIndex}-${row}-${col}" name="${Const.ParamsNames.FEEDBACK_QUESTION_RUBRICCHOICE}-${questionIndex}-${responseIndex}-${row}" value="${row}-${col}" ${checked}/><span class="color_neutral overlay"> ${description}</span>
+  <input class="overlay" type="radio" ${disabled} id="${Const.ParamsNames.FEEDBACK_QUESTION_RUBRICCHOICE}-${questionIndex}-${responseIndex}-${row}-${col}" name="${Const.ParamsNames.FEEDBACK_QUESTION_RUBRICCHOICE}-${questionIndex}-${responseIndex}-${row}" value="${row}-${col}" ${checked}/><span class="color_neutral overlay"> ${description}</span>
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionRubricSubmissionFormBodyFragment.html]: There is a `span` tag at the end of this line; it should be moved to the next line.

File: src/main/resources/feedbackQuestionRubricSubmissionFormMobilePanelFragment.html [hunk 35]
```diff
@@ -1,4 +1,4 @@
 <div class="radio">
-    <label><input type="radio" ${disabled} id="mobile-${Const.ParamsNames.FEEDBACK_QUESTION_RUBRICCHOICE}-${questionIndex}-${responseIndex}-${row}-${col}" name="mobile-${Const.ParamsNames.FEEDBACK_QUESTION_RUBRICCHOICE}-${questionIndex}-${responseIndex}-${row}" value="${row}-${col}" ${checked}/>
-        <strong>${rubricChoiceValue}</strong> - ${description}</label>
+  <label><input type="radio" ${disabled} id="mobile-${Const.ParamsNames.FEEDBACK_QUESTION_RUBRICCHOICE}-${questionIndex}-${responseIndex}-${row}-${col}" name="mobile-${Const.ParamsNames.FEEDBACK_QUESTION_RUBRICCHOICE}-${questionIndex}-${responseIndex}-${row}" value="${row}-${col}" ${checked}/>
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionRubricSubmissionFormMobilePanelFragment.html]: The `input` tag should be on the next line.

File: src/main/resources/feedbackQuestionRubricSubmissionFormMobilePanelFragment.html [hunk 36]
```diff
@@ -1,4 +1,4 @@
...
+    <strong>${rubricChoiceValue}</strong> - ${description}</label>
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionRubricSubmissionFormMobilePanelFragment.html]: The `</label>` closing tag should be on the next line.

File: src/main/resources/feedbackQuestionTextEditFormTemplate.html [hunk 37]
```diff
@@ -1,13 +1,13 @@
...
-                title="The recommended length is shown to the respondent but not enforced">
-                Recommended length</span> for the response: <input disabled
-                type="number" class="form-control"
-                name="recommendedlength" value="${recommendedlength}">
-            words
-        </div>
+  <br>
+  <div class="row">
+    <div class="col-xs-12 question-recommended-length">
+      [Optional] <span data-toggle="tooltip" data-placement="top"
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionTextEditFormTemplate.html]: This block looks weird. Tidy up?
```html
[Optional]
<span>
  Recommended length
</span>
for the response:
<input>
words
```

File: src/main/resources/feedbackQuestionTextSubmissionFormTemplate.html [hunk 38]
```diff
@@ -2,18 +2,18 @@
...
-    </div>
-    <div class="pull-right">
-        Response length:
-        <span id="responseLength-${questionIndex}-${responseIndex}">0</span>
-        words
-    </div>
+  <div class="col-md-6 padding-0" ${recommendedLengthDisplay}>
+    Recommended length for the answer:
+    <span id="recommendedLength-${questionIndex}-${responseIndex}">
+      ${recommendedLength} </span>
```
Comment (wkurniawan07) [inline src/main/resources/feedbackQuestionTextSubmissionFormTemplate.html]: The `</span>` closing tag should be on the next line.

**Comment by sujeet14108** [comment]
@wkurniawan07 Ready for review :)

**Comment by sujeet14108** [comment]
@wkurniawan07 Ready for review :)

**Comment by wkurniawan07** [comment]
Some of the email templates changes are still included.

**Comment by wkurniawan07** [comment]
Two email templates-related files still remain. In addition, there is a merge conflict that needs resolving.

**Comment by sujeet14108** [comment]
@wkurniawan07 You said 2 files are remaining one is `instructorEmailFragment-courseJoin.html
` . Can you tell me other

**Comment by wkurniawan07** [comment]
@sujeet14108 the fact that commit d3a10ab is necessary should tell you what the other file (or rather, two files) is (are).

**Comment by sujeet14108** [comment]
@wkurniawan07 Corrected both the PR's this and #7982 
Please have a look :)

**Pull request merged**